### ISSUE TITLE
Please help me make Goldeneye 007 work

### ISSUE THREAD

**Issue by AlexN64gamer** [ata4/angrylion-rdp-plus#120]
Goldeneye 007 (USA rom) is not working at all with the lastest 1.6 release, Ive tried filtered, unfiltered, Blocky, Blurry, etc, and the result is always "fatal error, stopping emulation" on project64 2.4 Ive tried other games with angrylion´s and they work flawless, so the problem is Goldeneye usa version, my specs are Intel I7 10700K 3,6 (8 cores 16 threads) MSI RTX 2060 6gb , 32 gb ram corsair vengeance ddr5, I always use the plugin on "slow", never had any FPS problems, what am I doing wrong?

**Comment by RokkumanX** [comment]
Project64 2.4 is rather old now, they have made a 3.0 release and even a 3.0.1 release which can be downloaded [here](https://www.pj64-emu.com/file/project64-3-0-1-5664-2df3434/)

Furthermore I would also like to recommend [m64p](https://github.com/loganmc10/m64p/releases) if it's accuracy you are after, m64p utilizes ParaLLEl-RDP for video and the ParaLLEl-RSP.

You can even use angrylion-rdp-plus and cxd4's RSP in m64p, more information on that can be found [here](http://emulation.gametechwiki.com/index.php/M64p) under the **Tips and tricks (Bonus) section**.

Although I would like to say that I have the PAL version and the game works flawless in both Project64 3.0.1 and m64p, could it be that you have a faulty dump of the USA version of the game, if that's the case then I suggest re-dumping the game and get these checksums:
`crc B6330846 md5 70C525880240C1E838B8B1BE35666C3B sha1 ABE01E4AEB033B6C0836819F549C791B26CFDE83`

**Comment by RokkumanX** [comment]
I would also like to point out that Project64 isn't even recommended to use anymore ([source here](http://emulation.gametechwiki.com/index.php/Nintendo_64_emulators)) if you look at the chart provided there.

Usually those charts represent a good and accurate view of the current emulation situation for each given system.

Project64 have had it's fair share of issues over the years and so have Mupen64Plus, but in my honest and humble opinion I do believe Mupen64Plus have surpassed Project64 in both accuracy and general usage since a good while now and m64p is a very very good proof of just that.

It's easy to use and focuses on accuracy only, the only good thing that could be said about Project64 is that it still have the ability to choose from different plugins easily from within the GUI.
But again there is ways of changing plugins in m64p as I described earlier above in my first post.

I know this doesn't help you in the way you want but GoldenEye 007 (version doesn't matter) shouldn't have any issues with angrylion-rdp-plus.

And fatal errors sounds more like the emulators fault then angrylion, again 2.4 is quite old now and was prone to cause issues.

Furthermore you never told us where you downloaded both the emulator or the plugin, let me just tell you right now that you should avoid EmuCR builds like the plague.
Nothing good have ever come from that site more than "emulation news"

Anyway, I have rambled on long enough.

tl;dr

Verify your copy of the game, alternatively switch emulator.

**Comment by RokkumanX** [comment]
I don't condone piracy in any way but I did this to prove that GoldenEye (USA) works with latest version of angrylion-rdp-plus and in latest 3.0.1 release of Project64.

![Proof](https://user-images.githubusercontent.com/85066613/123267168-0031a500-d4fd-11eb-8047-08b0caab05b1.png)

Forgive me Jesus....
